addons/bestja_frontend_ucw/static/src/js/custom.js

```diff
@@ -43,4 +43,41 @@ jQuery(window).ready(function () {
         });
     });
     /* End of smooth scrolling */
+
+    $('#cookie_reminder_close').click(function() {
+        $('#cookie_reminder_container').hide();
+    });
+
+
+    /* Cookie for reminder */
+    function setCookie(cookieName, cookieValue, daysToExpire) {
+        var d = new Date();
+        d.setTime(d.getTime() + (daysToExpire * 24 * 60 * 60 * 1000));
+        var expires = "expires=" + d.toUTCString();
+        document.cookie = cookieName + "=" + cookieValue + "; " + expires;
+    }
+
+    function getCookie(cookieName) {
+        var name = cookieName + "=";
+        var ca = document.cookie.split(';');
+        for (var i = 0; i < ca.length; i++) {
+            var c = ca[i];
+            while (c.charAt(0) == ' ')
+                c = c.substring(1);
+            if (c.indexOf(name) == 0)
+                return c.substring(name.length, c.length);
+        }
+        //console.log('Now there is no cookie by the name: ' + cookieName);
+        return "";
+    }
+
+    $('#cookie_reminder_close').click(function(){
+        setCookie('accepted','yes',7);
```

Comment:
Name “accepted” for that cookie is… unacceptable. :) It should be more verbose, like “cookie_info_dismissed”.